app/controllers/users_controller.rb

```diff
@@ -1,12 +1,31 @@
 # frozen_string_literal: true
 
+#  = User Controller
+#
+#  ==== Index
+#  index  == index_user ==list_users
+#  users_by_contribution
+#  users_by_name (admin only)
+#  user_search
```

Comment:
I need to review the routes, paths, action names, and aliases to make sure I've got it all correct.
I'd also like to keep it simple and conform to other controllers e.g., 
`:search` rather than `:user_search` (with appropriate changes in the routes, aliases, tests)

Reply:
I agree about :search action names. 
Will check through these PRs this weekend. I'm just glancing via the phone now, but hopefully we'll implement uniform index params across all the controllers too (for search and other index-refining and -sorting actions)